_posts/en/newsletters/2021-02-03-newsletter.md

```diff
@@ -0,0 +1,93 @@
+---
+title: 'Bitcoin Optech Newsletter #134'
+permalink: /en/newsletters/2021/02/03/
+name: 2021-02-03-newsletter
+slug: 2021-02-03-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to a blog post about how a small change to
+the Script language after taproot activation could enable increased
+contract flexibility and includes our regular sections with brief
+descriptions of releases, release candidates, and notable changes to
+popular Bitcoin infrastructure projects.
+
+## News
+
+- **Replicating `OP_CHECKSIGFROMSTACK` with BIP340 and `OP_CAT`:** Andrew
+  Poelstra authored a [blog post][poelstra 340cat] about implementing
+  the functionality of the
+  [OP_CHECKSIGFROMSTACK][topic op_checksigfromstack] (`OP_CSFS`) opcode
+  from [ElementsProject.org][] on Bitcoin using the proposed [BIP340][] specification of [schnorr
+  signatures][topic schnorr signatures] and an [OP_CAT][csfs cat] opcode
+  that was part of Bitcoin until mid-2010 (and which is often mentioned
+  as a candidate for reintroduction).  Enabling CSFS-like behavior on
+  Bitcoin would allow the creation of [covenants][topic covenants] and
+  other advanced contracts without having to presign spending
+  transactions, possibly reducing complexity and the amount of data that
+  needs to be stored.  The post ends with a teaser for later posts in
+  the series (links added by us):
+
+    > "In our next posts, we’ll talk about how to use auxiliary inputs
+    > to simulate [SIGHASH_NOINPUT][topic sighash_noinput] and enable
+    > constant-sized backups for Lightning channels, and how to use
+    > "value-switching" to construct [vaults][topic vaults].  In our
+    > final post we’ll talk about ad-hoc extensions of
+    > [Miniscript][topic miniscript], and how to develop software for
+    > these constructions in a
+    > maintainable way."
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [BTCPay Server 1.0.6.8][] is a release attempting to mitigate Google Safe
+  Browsing's [false positive flagging of BTCPay instances][BTCPay Server #2139] as
+  "deceptive sites". The issue occurred across different domains and web hosts
+  and even on private websites, hinting that content or structure of the pages
+  were the trigger. A series of speculative commits including removing the Tor
+  URL from the login page and removing clipboard and transparency overlay
+  features were made to try to prevent the flagging.
```

Comment:
Why recommend this update if it clearly didn't work? I think it'd make more sense to say "don't bother".

Reply:
Reference: https://github.com/btcpayserver/btcpayserver/issues/2139#issuecomment-770202463

I suggest we just remove this item (and the "Releases and release candidates" this week, since this is the only item).